packages/appstore/src/modules/traders-hub/account-with-wallets.tsx

```diff
@@ -0,0 +1,117 @@
+import { observer, useStore } from '@deriv/stores';
+import Wallet from 'Components/wallet';
+import React from 'react';
+
+// TODO: delete it after testing
+type TProps = {
+    show_test_wallets?: boolean;
+};
+
+const AccountWithWallets = observer(({ show_test_wallets = false }: TProps) => {
+    const {
+        client: { accounts },
+    } = useStore();
+
+    const wallet_test_accounts: typeof accounts[0][] = [
```

Comment:
since we have real API, let's remove these dummy wallets and show real ones. Needs adjustments in the client store

Reply:
@mahdiyeh-deriv now on QA29 we have only accounts just with fiat wallets, but I have to show crypto wallets too. I left a select to hide the test wallets. Also, we still can't create even MT5 account on QA29 (image attached). So I guess we need these test accounts for now.

<img width="489" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/915f954e-ff1a-4f8a-9ffd-c425387e433a">

Reply:
@sergei-deriv yes, yes, we need to update it with the real API later on